sdcm/sct_events/database.py

```diff
@@ -381,6 +381,13 @@ def __init__(self, node: str, severity=Severity.NORMAL, **__):
         super().__init__(node=node, severity=severity, publish_event=True)
 
 
+class ScyllaYamlUpdateEvent(ScyllaDatabaseContinuousEvent):
```

Comment:
I don't consider it as continuous event - changing yaml file is quite atomic.

Reply:
So we have 2 votes to 1 on the Info vs. Continuous event debate :) I concede and moved the event to being a `InformationalEvent`